acceptance-tests/lib/constants.js

```diff
@@ -1,5 +1,5 @@
 const CONFIG_FILE_NAME = 'hs-acceptance-test.config.yml';
-const DEFAULT_CLI_PATH = '../packages/cli/bin/hs';
+const DEFAULT_CLI_PATH = '../bin/hs';
```

Comment:
This `const` was removed and now an inline variable is used [here](https://github.com/HubSpot/hubspot-cli/blob/main/acceptance-tests/lib/env.ts#L55)